File: 02_ProgrammingFundamentals/_05_Lists/_01_Lecture/_06_ListOfProducts.java
package _02_ProgrammingFundamentals._05_Lists._01_Lecture;

import java.util.ArrayList;
import java.util.Collections;
import java.util.List;
import java.util.Scanner;

public class _06_ListOfProducts {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read input value
        int n = Integer.parseInt(scan.nextLine());

        // Add variable
        List<String> productsList = new ArrayList<>();

        // Create list
        for (int i = 0; i < n; i++) {
            productsList.add(scan.nextLine());
        }

        // Arrange list
        Collections.sort(productsList);

        // Print result
        for (int i = 0; i < productsList.size(); i++) {
            System.out.printf("%d.%s%n", i + 1, productsList.get(i));
        }

        scan.close();
    }
}

File: 24_C#Fundamentals/_05_Lists/_01_Lecture/_06_ListOfProducts/_06_ListOfProducts.cs
using System;

public class _06_ListOfProducts
{
    public static void Main(string[] args)
    {
        // Read input value
        int n = int.Parse(Console.ReadLine());

        // Add variable
        List<string> productsList = new List<string>();

        // Create list
        for (int i = 0; i < n; i++)
        {
            productsList.Add(Console.ReadLine());
        }

        // Arrange list
        productsList.Sort();

        // Print result
        for (int i = 0; i < productsList.Count; i++)
        {
            Console.WriteLine($"{i + 1}.{productsList[i]}");
        }
    }
}
